### DISCUSSION TITLE
Version stable 3.4.3

### DISCUSSION THREAD

**Discussion by ynaessens** [JeromeDevome/GRR#175] [General]
Cette version apporte plusieurs corrections de code et deux ajouts :
- la prise en compte des adresses iPv6
- la fonctionnalité "participants": les utilisateurs autorisés peuvent lors de la définition d'une réservation, ouvrir les inscriptions aux utilisateurs connectés, qui s'inscrivent alors via la page de visualisation de la réservation. 

<hr /><em>This discussion was created from the release <a href='https://github.com/JeromeDevome/GRR/releases/tag/v.3.4.3'>Version stable 3.4.3</a>.</em>

**Comment by ividalay** [comment]
Bonjour,
Pouvez-vous me donner la version de php nécessaire à la version 3.4.3 de GRR ?
Merci,
Cordialement,
IVA

**Reply by ynaessens** [reply]
Bonjour,
c'est indiqué dans le fichier README : https://github.com/JeromeDevome/GRR/blob/GRR-3.4.3/README.md
Cordialement,
YN

**Comment by Coy0te** [comment]
Bonjour,
Est-il possible de passer d'une 3.3.0 à cette version, sans casser la base de données existante ?
Si oui, comment ? La doc ne mentionne a priori que l'installation, pas la mise à jour d'une version à l'autre. 
Merci,

**Reply by ynaessens** [reply]
Bonjour,
c'est indiqué dans le fichier INSTALL.txt.
Il n'y a pas de souci à la montée de version. Cependant, s'il vous venait à redescendre, la nouvelle structure de la base de données pourrait poser des problèmes. D'où le conseil de faire une sauvegarde avant de monter en version.
Notez qu'une sous-version apportant quelques correctifs vient de sortir.
Cordialement,
YN
